ClientApp/src/components/Nav/NavMenu.css

```diff
@@ -61,7 +61,7 @@ fill: rgba(0, 0, 0, 0.5);
 
 .af-navmenu-brand {
     /* sam: *makes his sexy logo as large as possible* */
-    height: 1.8rem;
+    height: 2.4rem;
```

Comment:
In future, please include an accompanying sarcastic comment whenever changing this value.

Reply:
My sincerest apologies dear scrumlordiam samiam.